<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RscManagement</class>
 <widget class="QDialog" name="RscManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ressourcenmanagement</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="8" column="1">
        <widget class="QPushButton" name="buttonAddCat">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Neue Kategorie</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="buttonAddResource">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Neue Ressource</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="buttonDelCat">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Kategorie löschen</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Los!</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditSearch"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Anzeige nach:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Suche nach</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="3">
        <widget class="ResourceTree" name="treeWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">3</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">4</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="description">
         <property name="text">
          <string>Beschreibung</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="type">
         <property name="text">
          <string>Typ</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBoxSetFilter">
         <item>
          <property name="text">
           <string>Kategorie</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Raum</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QPushButton" name="buttonDelResource">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ressource löschen</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QPushButton" name="buttonDelRoom">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Raum löschen</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPushButton" name="buttonAddRoom">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Neuer Raum</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="make">
         <property name="text">
          <string>Hersteller</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelEquipmentnumber">
          <property name="text">
           <string>Ausstattungsnummer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditEquipmentnumber">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelActivationDate">
          <property name="text">
           <string>Aktivierungsdatum</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDateEdit" name="dateEditActiveDate">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelCostLocation">
          <property name="text">
           <string>Kostenstelle</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="labelRoom">
          <property name="text">
           <string>Raum</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="comboBoxRoom">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="labelCategory">
          <property name="text">
           <string>Kategorie</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QComboBox" name="comboBoxCategory">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="29" column="0">
         <widget class="QLabel" name="labelMinQty">
          <property name="text">
           <string>Minimale Menge</string>
          </property>
         </widget>
        </item>
        <item row="30" column="1">
         <widget class="QPushButton" name="buttonSubmitResourceChanges">
          <property name="text">
           <string>Änderungen übernehmen</string>
          </property>
         </widget>
        </item>
        <item row="29" column="1">
         <widget class="QSpinBox" name="spinBoxMinQty">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="labelLocker">
          <property name="text">
           <string>Schrank</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="comboBoxLocker">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="labelType1">
          <property name="text">
           <string>Hersteller</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QComboBox" name="comboBoxMake">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="labelType2">
          <property name="text">
           <string>Typ</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLineEdit" name="lineEditType">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="labelDescription">
          <property name="text">
           <string>Beschreibung</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QTextEdit" name="textEditDescription">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QLineEdit" name="lineEditSerialnumber">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="labelSerialnumber">
          <property name="text">
           <string>Seriennummer</string>
          </property>
         </widget>
        </item>
        <item row="22" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxPurchasePrice">
          <property name="maximum">
           <double>250000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="22" column="0">
         <widget class="QLabel" name="labelPurchasePrice">
          <property name="text">
           <string>Einkaufspreis</string>
          </property>
         </widget>
        </item>
        <item row="25" column="1">
         <widget class="QSpinBox" name="spinBoxTotalQty">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="25" column="0">
         <widget class="QLabel" name="labelTotalQty">
          <property name="text">
           <string>Gesamtmenge</string>
          </property>
         </widget>
        </item>
        <item row="27" column="1">
         <widget class="QSpinBox" name="spinBoxQtyInStock">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="27" column="0">
         <widget class="QLabel" name="labelQtyInStock">
          <property name="text">
           <string>Lagernde Menge</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxCostLoc">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="labelCatSelected">
          <property name="text">
           <string>Kategorie ausgewählt</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineEditCatName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelCatName">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="buttonSubmitChangesCat">
          <property name="text">
           <string>Änderungen übernehmen</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="labelRoomSelected">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Raum ausgewählt</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelRoomName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Raumname:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditRoomName">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonSubmitChangesRoom">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Änderungen übernehmen</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResourceTree</class>
   <extends>QTreeWidget</extends>
   <header>resourcetree.h</header>
   <slots>
    <slot>updateTree()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RscManagement</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>934</x>
     <y>735</y>
    </hint>
    <hint type="destinationlabel">
     <x>976</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
